### PR TITLE
[Patch] v1.6.1 release base

### PR DESCRIPTION
**Pull request by wild-endeavor**
v1.6.1 patch release.  Please see the release notes for changes.

### CODE REVIEW

File: docker/sandbox-bundled/manifests/complete.yaml [hunk 1]
```diff
@@ -403,28 +403,15 @@ metadata:
...
-      level: 6
-    propeller:
-      create-flyteworkflow-crd: true
-    webhook:
-      certDir: /var/run/flyte/certs
-      localCert: true
-      secretName: flyte-sandbox-webhook-secret
-      serviceName: flyte-sandbox-webhook
-      servicePort: 443
+  000-core.yaml: "admin:\n  endpoint: localhost:8089\n  insecure: true\ncatalog-cache:\n
```
Comment (jeevb) [inline docker/sandbox-bundled/manifests/complete.yaml:406]: Why the change in rendering here?

Comment (wild-endeavor) [reply docker/sandbox-bundled/manifests/complete.yaml:406]: no idea. this isn't me, this is make helm

**Review by eapolinario — APPROVED** [review]

**Review by eapolinario — DISMISSED** [review]

**Review by eapolinario — APPROVED** [review]
